acos_client/tests/t.py

```diff
@@ -214,7 +224,12 @@ def run_all(ax, partition, pmap):
     print("")
     print("About to do a close with bad session id")
     c.session.session_id = 'bad_session_id'
-    c.session.close()
+    try:
+        c.session.close()
+    except acos_client.errors.InvalidSessionID:
+        print("got invalid session error, good")
```

Comment:
Same as above. We don't wrap that in a `try` to begin with because we want to see exceptions raised as the session should close without issue.

Reply:
I cleaned this up and rebased with PR #228